x/bank/keeper/send.go

```diff
@@ -96,6 +96,15 @@ func (k BaseSendKeeper) InputOutputCoins(ctx sdk.Context, inputs []types.Input,
 				sdk.NewAttribute(sdk.AttributeKeyAmount, out.Coins.String()),
 			),
 		)
+
+		// Create account if recipient does not exist.
```

Comment:
Can we please add a bug fix changelog entry for this?

Reply:
I also added a `State Machine Breaking` changelog entry for it. Does it make sense?

Reply:
Yes, thank you!